cli/dcoscli/data/help/job.txt

```diff
@@ -12,13 +12,13 @@ Description:
         dcos job update <job-file>
         dcos job kill <job-id> [<run-id>][--all]
         dcos job run <job-id> [--json]
-        dcos job list [--json]
+        dcos job list [--json --quiet]
```

Comment:
Should `--json` and `--quiet` be mutually exclusive? As this doesn't make sense to use them together.

Reply:
definitely!